lib/signs/bus.ex

```diff
@@ -165,10 +165,10 @@ defmodule Signs.Bus do
       |> Enum.group_by(&{&1.stop_id, &1.route_id, &1.direction_id})
 
     # Compute new sign text and audio
-    {[top, bottom], audios} =
+    {[top, bottom], audios, tts_audios} =
       cond do
         config == :off ->
-          {["", ""], []}
+          {["", ""], [], []}
```

Comment:
This complex data type formed of anonymous lists and tuples is becoming a bit unwieldy, I think. Is it time to introduce a struct with named fields (and, ideally, a typespec)?

Reply:
I think a struct would be too heavy here, since these values are just being returned to the outer scope and then immediately used. I added some types to document things a bit, and also added some explicit variables to this line (which otherwise looks particularly odd because of the empty values)

Reply:
Nice, the type definitely helps.

We could also keep the variable approach concise like this, if you want:

```elixir
{_messages = ["", ""], _audios = [], _tts_audios = []}
```

Reply:
Oh interesting, I didn't realize that was possible. Thanks.